x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -28,22 +28,31 @@ import { useTestIdGenerator } from '../../../../hooks/use_test_id_generator';
 import { useDataTestSubj } from '../../hooks/state_selectors/use_data_test_subj';
 
 const CommandInputContainer = styled.div`
+  background-color: ${({ theme: { eui } }) => eui.euiColorGhost};
```

Comment:
Looks like a bad merge here. remove this line (31). the correct entry is below on line 32